demo.ipynb

```diff
@@ -10,6 +10,94 @@
         "<a href=\"https://colab.research.google.com/github/geraschenko/gmrfs/blob/main/demo.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
       ]
     },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Fitting a latent 2d Gaussian Markov Random Field\n",
```

Comment:
What makes it 2d? Isn't this model applicable to any graph, regardless of whether it's planar?

Reply:
Addressed in https://github.com/geraschenko/gmrfs/pull/2/commits/cdc292c0fd70e73ce3d56a5d2b26bd4402382e31

Reply:
You are right, it is not specific to 2d. We should remove that comment. Sorry.